doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -594,7 +594,9 @@ Security libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`modem_key_mgmt` library:
```

Comment:
Please, remove the changelog entry from this and add it as a suggestion to one of my PRs. See my mail.